src/modules/enhancedPOI/docs/sv_SE.md

```diff
@@ -0,0 +1,11 @@
+Att placera intressepunkter är nödvändigt för att många meddelanden ska kunna genereras. Om du inte vill använda andra spelares POI eller av någon annan anledning placera din egen IP kan det vara en tråkig aktivitet.
+
+::: danger Mapkit
+På grund av begränsningarna i Mapkit är det för närvarande inte möjligt att använda den här modulen om du använder karttypen `Mapkit`.
+:::
+
+För att göra placeringen enklare erbjuder denna modul följande förenklingar:
+* Alla befintliga intressepunkter av den valda typen är färgade.
+* Om markören rör sig utanför bildens räckvidd centreras kartan automatiskt på markören.
+
+![Exempel: Bos](/v4/docs/tillgångar/förbättradPOI/img/förbättradPOI_SV.png)
```

Comment:
`assets/enhancedPOI`  instead of `tillgångar/förbättradPOI`. The images are still missing but we can create them when the translations are online.